lib/runtime/outline.js

```diff
@@ -18,7 +18,7 @@ export function activate (ink) {
   subs.add(client.onDetached(() => pane.setItems([])))
 
   let edSubs = new CompositeDisposable()
-  subs.add(atom.workspace.onDidChangeActiveTextEditor(ed => {
+  subs.add(atom.workspace.onDidChangeActiveTextEditor(throttle(ed => {
```

Comment:
I think `throttle` here would be reasonable, and won't make Julia hard-work when consecutive editor switches.

Reply:
Yeah, good call.